### ISSUE TITLE
[HW2] Api 관련 질문드립니다.

### ISSUE THREAD

**Issue by sharkmir1** [swsnu/swppfall2019#67]
PUT api/user/1을 할 경우 해당 object를 완전히 새로운 object로 대체하는 것이 비효율적인 것 같아서, 
PATCH를 통해 logged_in 값만 바꾸도록 구현하였습니다. spec 상에 언급되어 있지 않은 부분이라, PUT 대신 PATCH를 사용해도 되는지 알고싶습니다!

**Comment by ktaebum** [comment]
yes you can use `PATCH`. We will just check end to end behavior :slightly_smiling_face:
